docs/nodes/maintain/chain-config-flags.md

```diff
@@ -76,23 +76,44 @@ adversely affect your node operation.
 
 ### State Sync
 
-For example, as of [AvalancheGo v1.7.17
-(Verbier)](../../apis/avalanchego/avalanchego-release-notes.md#v1717---verbier-view-on-github),
-the default for [State Sync](#state-sync-enabled-boolean) is false. In order to
-enable it, you can add the following to `{chain-config-dir}/C/config.json`
+#### `state-sync-enabled` (boolean)
```

Comment:
This is a duplicate of later part of this doc: https://docs.avax.network/nodes/maintain/chain-config-flags#state-sync-settings. We should consolidate these two into one.